script.playrandomvideos/python/context.py

```diff
@@ -0,0 +1,14 @@
+import sys
+import xbmc
+
+from lib import playrandom
+from lib.pykodi import get_pathinfo
+
+if __name__ == '__main__':
+    path = sys.listitem.getPath()
```

Comment:
Can you explain how the listitem gets into sys?

Reply:
Magic that Kodi does for context item add-ons. I'm not sure how it works internally but it has a quck mention in the wiki, section 2, https://kodi.wiki/view/Context_Item_Add-ons .

Reply:
Thanks